core/arch/x86_64/mm/core_mmu.c

```diff
@@ -1025,7 +1025,6 @@ static unsigned long init_mem_map(struct tee_mmap_region *memory_map,
 	 */
 	vaddr_t id_map_start = (vaddr_t)__identity_map_init_start;
 	vaddr_t id_map_end = (vaddr_t)__identity_map_init_end;
-	unsigned long offs = 0;
```

Comment:
Can we keep this offs variable in init_mem_map?

Reply:
Why keep it? Actually, it is useless and result in compile warning.

Reply:
Because we may need to implement ALSR for X86 as well. What kind of compile warning did you meet?

Reply:
The warning is： 
core/arch/x86_64/mm/core_mmu.c: In function ‘init_mem_map’:
core/arch/x86_64/mm/core_mmu.c:1029:16: warning: variable ‘offs’ set but not used [-Wunused-but-set-variable]
 1029 |  unsigned long offs = 0;

We can add it back when implement ALSR.

Reply:
Done, restore offs